app/commands/v2/put_content.rb

```diff
@@ -304,8 +307,13 @@ def raise_if_links_is_provided
         )
       end
 
-      def validate_schema
-        SchemaValidator.new(type: :schema).validate(payload.except(:content_id))
+      def schema_validation_errors
+        @_validation_errors ||=
```

Comment:
ah right. it's a reasonably widely-used convention that says "this is an instance variable that is only used to avoid repeating a service/db call, rather than representing some information that's relevant to the domain". but if it's not used elsewhere in the codebase, that might indicate that i should avoid it!